profiles/admin.py

```diff
@@ -19,3 +19,8 @@ class ProfileAdmin(VersionAdmin):
     search_fields = ("user__username", "github_account", "user__email", "email")
     list_display = ("github_account", "email", username, user_email)
     raw_id_fields = ["user"]
+
+
+@admin.register(ExtraFields)
+class ExtraFieldsAdmin(VersionAdmin):
+    list_select_related = ["profile"]
```

Comment:
Let's add `raw_id_fields` or we'll have a dropdown in the admin of ~18k users. 

```suggestion
    list_select_related = ["profile"]
    raw_id_fields = ["profile"]
```